01week/resume/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta name="format-detection" content="telephone=no">
+    <link rel="stylesheet" href="style.css">
+    <title>Andrew Hartford - Resume</title>
+  </head>
+
+  <body>
+    <p id="name">Andrew Hartford</p>
+    <p id="descrip">Developer in Austin, Texas</p>
+    <h3 id="contact">andrewhartford@utexas.edu | (210) 852-0750</h3>
+
+    <div class="Objective">
```

Comment:
Html uses spinal-case and gets upset when you use upper-cases. Just uncapitalize the O here.